pensieve/pensieve.py

```diff
@@ -353,18 +380,20 @@ def search_for_images(self):
 
 
 if __name__ == '__main__':
-    try:
-        harry_pars = pickle.load(open('harry_pars_full_corpus.pkl', 'rb'))
-    except IOError:
-        corpus = Corpus('../corpus/')
-        harry_pars = corpus.find_character_paragraphs(['Harry', 'Potter'], 0.4)
-        pickle.dump(harry_pars, open('harry_pars_full_corpus.pkl', 'wb'))
-
-    from pprint import pprint
-    print('{} paragraphs found'.format(len(harry_pars)))
-    rand_index = randint(0, len(harry_pars)-1)
-    test_par = harry_pars[rand_index]
-    print('Book '+str(test_par.doc.id+1))
-    print('Paragraph '+str(test_par.id))
-    print(test_par.text)
-    pprint(test_par.words)
+    # try:
```

Comment:
Should consider getting rid of the commented out code....but again not necessary